### DISCUSSION TITLE
Welcome to jetstrap Discussions!

### DISCUSSION THREAD

**Discussion by tsommie** [nascent-africa/jetstrap#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by luisprmat** [comment]
Thanks @tsommie, i'm a Colombian Laravel developer, I like to contribute with several open-source packages for Laravel, I like bootstrap css framework over Tailwind.
I ask me why did you drop support for Bootstrap 5 when it was present in the first version of this package.

**Reply by tsommie** [reply]
I felt it was an overkill considering:

1. It had no RC
2. The presets (AdminLTE and CoreUI) didn't support it yet
3. Migrating from 4 to 5 was easier than managing both versions in the package.

**Reply by tsommie** [reply]
Thanks for your contribution to the package, I appreciate it. And am a Laravel developer from Nigeria.

**Reply by REJack** [reply]
I will start with AdminLTE v4 (Bootstrap 5 based Version) soon 😄

**Reply by tsommie** [reply]
> I will start with AdminLTE v4 (Bootstrap 5 based Version) soon

Can't wait...

**Reply by ishratali1997** [reply]
how to use any other admin panel template?

**Reply by RobinBastiaan** [reply]
What is the estimated time for the arrival of an update of this package to Bootstrap 5. Should I wait using this package till then, or are there other ways around it in the mean time? Thanks!

**Reply by tsommie** [reply]
The package already supports Bootstrap 5 if your using the defaults, However, 4 is used if you're using CoreUi or AdminLTE.

**Reply by RobinBastiaan** [reply]
Right, I see now. The right side bar project description got me confused. Thanks for all the good work!

**Reply by calebichen** [reply]
> I will start with AdminLTE v4 (Bootstrap 5 based Version) soon 😄

Look forward to the new version.

**Comment by camilodw** [comment]
hi I need to install jetstrap with bootstrap 4 but I can't find any clear documentation on how to do it

**Reply by thantzin-soe** [reply]
@camilodw 
You have to install version 2.4 of jetstrap because the latest jetstrap version generates the stubs with bootstrap 5 as default.

**Comment by spitfire64** [comment]
Hi, I am new to Jetstream and I appreciate this project very much since I can focus on jetstream and let go of tailwind. Could someone please show some code of a simple Crud example using this project's app.blade.php and its jetstream components? Such an example should get me going.
Thanks, and cheers from Sweden!
